tui/assets/ansi.py

```
@@ -0,0 +1,62 @@
+import re
```

Ruff (0.16.2)

[warning] The file is executable but no shebang is present

(EXE002)